Bottom: d9f3b03b646fa5dc298f97d21aaba3d44f4ea6de
Top:    eb232ae54c99e18c87fc1a3d6d18b0a9266a449e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 04:07:25 +0000

Disable debug


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 56dddb0..393467b 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1514,10 +1514,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
 //  gui->invalidate_all ();
 
   /* ASSUME THERE IS A POLYGON WHERE WE KNOW WE PUT ONE... */
-  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
-  gui->invalidate_all ();
+//  ((PolygonType *)PCB->Data->Layer[1].Polygon->data)->Clipped = info.poly;
+//  gui->invalidate_all ();
 
-//  poly_Free (&info.poly);
+  poly_Free (&info.poly);
 
   return objects;
 }
